packages/gatsby/src/redux/__tests__/pages.js

```diff
@@ -3,8 +3,18 @@
 const glob = require(`glob`)
 const reducer = require(`../reducers/pages`)
 const { actions } = require(`../actions`)
+const { readFile } = require(`fs-extra`)
```

Comment:
@chuntley I would suggest to use `memory-fs` instead of `fs-extra`:

```js
import MemoryFs from 'memory-fs';

const mockFs = jest.fn(() => new MemoryFs());

jest.doMock('fs', mockFs);

const fs = require('fs');

// ...
```

In my view, the code will be more clean.

Reply:
Sounds like a nice addition if you'd like to add it! As it is, let's not block this PR in the meantime.

Reply:
@KyleAMathews Ok ;)